spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -346,6 +346,44 @@
           inline_error = strip_tags(t('doc_auth.errors.alerts.selfie_not_live_poor_quality'))
           expect(page).to have_content(inline_error)
         end
+        # New test
+        it 'try again and page show selfie fail inline error message' do
+          visit_idp_from_oidc_sp_with_ial2
+          sign_in_and_2fa_user(user)
+          complete_doc_auth_steps_before_document_capture_step
+          attach_images(
+            Rails.root.join(
+              'spec', 'fixtures',
+              'ial2_test_portrait_match_failure.yml'
+            ),
+          )
+          attach_selfie(
+            Rails.root.join(
+              'spec', 'fixtures',
+              'ial2_test_portrait_match_failure.yml'
+            ),
+          )
+          submit_images
+          message = strip_tags(t('errors.doc_auth.selfie_fail_heading'))
+          expect(page).to have_content(message)
+          detail_message = strip_tags(t('doc_auth.errors.alerts.selfie_poor_quality'))
+          security_message = strip_tags(
```

Comment:
I'm not sure security message is an accurate name for this. Perhaps `warning_message`?

Reply:
Agreed, but that would mean renaming it several places in this file and I'd prefer to avoid that for this PR.